sources/packages/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -283,4 +284,68 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
   async getOfferingById(offeringId: number): Promise<EducationProgramOffering> {
     return this.repo.findOne(offeringId);
   }
+
+  /**
+   * Get programs for a particular institution in paginated.
+   * @param institutionId id of the institution.
+   * @param pageSize is the number of rows shown in the table
+   * @param skip is the number of rows that is skipped/offset from the total list.
+   * For example page 2 the skip would be 10 when we select 10 rows per page.
+   * @param sortColumn the sorting column.
+   * @param sortOrder sorting order default is descending.
+   * @param searchProgramName Search the program name in the query
+   * @returns programs, locations and offerings count, programs count under the specified institution.
+   */
+  async getPaginatedProgramsForInstitution(
+    institutionId: number,
+    pageSize?: number,
+    page?: number,
```

Comment:
to this API does the page would be like 0,1,2......n ?

Reply:
and this is equivalent to 1,2,...............(n+1) in UI right?

Reply:
yes @dheepak-aot